websites/S/Soundgasm/metadata.json

```
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "623396579960946690",
+		"name": "bear_"
+	},
+	"service": "Soundgasm",
+	"description": {
+		"en": "The future home for your audio hosting needs."
+	},
+	"url": [
+		"soundgasm.net",
+		"www.soundgasm.net"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/QIxjVEX.png",
+	"thumbnail": "https://i.imgur.com/3N7KG10.png",
+	"color": "#003399",
+	"category": "other",
+	"tags": [
+		"audio",
+		"asmr",
+		"music",
+		"recording",
+		"nsfw"
+	],
+	"settings": [
+		{
+			"id": "lang",
+			"multiLanguage": true
```

Since you use `youtube` [strings](https://docs.premid.app/en/dev/presence/metadata#multilanguage), this should be set to `"youtube"`
```suggestion
			"multiLanguage": "youtube"
```